### DISCUSSION TITLE
Github copilot signin issues with IntelliJ

### DISCUSSION THREAD

**Discussion by cah-shreya-kumar** [community/community#72700] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Issue :
copilot plugin version 1.3.3.3572

Sign in failed. Reason: Request signInInitiate failed with message: 980E0000:error:0A000152:SSL routines:final_renegotiate:unsafe legacy renegotiation disabled:C:\Users\runneradmin\AppData\Local\Temp\pkg.24e0b2b2d51e47b9dba34c30\node\deps\openssl\openssl\ssl\statem\extensions.c:908: , request id: 3, error code: -32603

When we tried to use a Proxy , we get the below error

Sign in failed. Reason: Request signInInitiate failed with message: tunneling socket could not be established, cause=connect ECONNREFUSED 127.0.0.1:10809, request id: 3, error code: -32603

**Comment by Akash1134** [comment]
Hi @cah-shreya-kumar ,

The error message that you seem to be encountering is due to your internal network connectivity.

Please confirm if your are behind a proxy setup and also test the copilot outside the proxy network to know if the issue persist.

To troubleshoot this issue further, refer the [Copilot network troubleshooting documentation](https://docs.github.com/en/copilot/troubleshooting-github-copilot/troubleshooting-network-errors-for-github-copilot#troubleshooting-proxy-errors). 

Thanks!

**Comment by drewjoh** [comment]
I'm getting this error also.  I'm not using a proxy.  

I'm on the latest PhpStorm, using nightly builds of the Copilot plug-in.  Copilot has been unusable for me for a while now.  I keep assuming my error reports are going to the right place and it will get resolved in an update... but I just updated to the latest nightly build with still no resolution. 😞 

![286303323-02dca6bf-352a-4e84-a741-d0e7aff1ffb9](https://github.com/community/community/assets/66233/ed4204c2-68b4-423d-b69a-ac5faff23044)

![286303834-180fdae2-c811-479b-9995-de6d46f1ec6f](https://github.com/community/community/assets/66233/cde38e60-0975-43a3-a423-747eff2e9c7f)

**Reply by drewjoh** [reply, marked as answer]
UPDATE:  In a last, desperate attempt, I just deleted my entire PhpStorm app and all plug-ins, reinstalled PhpStorm, reinstalled Copilot Plugin (latest nightly) and it's working as expected now!

I reinstalled via JetBrains Toolbox, in case that's a detail of importance.
